homeassistant/components/tplink/number.py

```diff
@@ -0,0 +1,91 @@
+"""Support for TPLink number entities."""
```

Comment:
We can strip this out before merging and put it in a second PR

Reply:
Yes, this PR is just for development use, the final one will be split up, and the new platforms moved into separate PRs.